main.js

```diff
@@ -80,14 +99,20 @@ function makeScreenFlex(cssTag) {
     document.querySelector(cssTag).style.display = 'flex';
 }
 
+function makeScreenGrid(cssTag) {
```

Comment:
are we still using this, i thought we weren't gonna use grid?

Reply:
Good spot!